app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,34 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
```

Comment:
- [ ] Based on requirements, you should follow the provided design. I noticed that the layout: composition and space between elements is completely different. 

Your design  | Original design
------------- | -------------
<img width="385" alt="Screenshot 2022-06-03 at 22 15 26" src="https://user-images.githubusercontent.com/77038610/171922677-81e576e4-01e0-4ba2-8ef4-c471a0f9eaee.png"> | <img width="277" alt="Screenshot 2022-06-03 at 22 15 52" src="https://user-images.githubusercontent.com/77038610/171922728-7c5632a6-0c36-4f7e-adbd-6257a815d098.png">

Kindly try to make the login and sign up pages look maximum similar 🙏🏻